staging/src/k8s.io/apiserver/pkg/util/flowcontrol/bootstrap/default-objects.go

```diff
@@ -0,0 +1,283 @@
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/user"
+)
+
+// The objects that define an apiserver's initial behavior
+var (
+	// InitialPriorityLevelConfigurations lists exempt first, default second
+	InitialPriorityLevelConfigurations = []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		DefaultPriorityLevelConfigurationDefault,
+	}
+	InitialFlowSchemas = []*rmv1a1.FlowSchema{
+		DefaultFlowSchemaExempt,
+		DefaultFlowSchemaDefault,
+	}
+)
+
+// Initial PriorityLevelConfiguration objects
+var (
+	DefaultPriorityLevelConfigurationExempt = pl(
+		rmv1a1.PriorityLevelConfigurationNameExempt,
+		nil)
+
+	DefaultPriorityLevelConfigurationDefault = plq(
+		rmv1a1.PriorityLevelConfigurationNameCatchAll, 100,
+		&rmv1a1.QueuingConfiguration{
+			Queues:           128,
+			HandSize:         6,
+			QueueLengthLimit: 100,
+		})
+)
+
+// Initial FlowSchema objects
+var (
+	DefaultFlowSchemaExempt = NewFSAllGroups(
+		rmv1a1.FlowSchemaNameExempt,
+		rmv1a1.PriorityLevelConfigurationNameExempt,
+		1, "",
+		user.SystemPrivilegedGroup,
+	)
+	DefaultFlowSchemaDefault = NewFSAllGroups(
+		rmv1a1.FlowSchemaNameCatchAll,
+		rmv1a1.PriorityLevelConfigurationNameCatchAll,
+		math.MaxInt32, rmv1a1.FlowDistinguisherMethodByUserType,
+		user.AllUnauthenticated, user.AllAuthenticated,
+	)
+)
+
+// PredefinedPriorityLevelConfigurations returns the instances of
+// the ones defined in the KEP
+// with the exempt one appearing first
+func PredefinedPriorityLevelConfigurations() []*rmv1a1.PriorityLevelConfiguration {
+	return []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		spl(plq("system", 30, &rmv1a1.QueuingConfiguration{
+			Queues:           1,
+			QueueLengthLimit: 1000})),
+		spl(plq("workload-high", 30, &rmv1a1.QueuingConfiguration{
+			Queues:           128,
+			HandSize:         6,
+			QueueLengthLimit: 100})),
+		spl(plq("workload-low", 30, &rmv1a1.QueuingConfiguration{
+			Queues:           128,
+			HandSize:         6,
+			QueueLengthLimit: 100})),
+		DefaultPriorityLevelConfigurationDefault,
+	}
+}
+
+var (
+	verbAll       = []string{rmv1a1.VerbAll}
+	apiGroupAll   = []string{rmv1a1.APIGroupAll}
+	apiGroupCore  = ""
+	nsAll         = []string{"c s", "*"}
+	nsSystem      = []string{"kube-system"}
+	resourceAll   = rmv1a1.ResourceAll
+	ruleRscAll    = []rmv1a1.ResourcePolicyRule{resourceRule(verbAll, apiGroupAll, nsAll, resourceAll)}
+	ruleNonRscAll = []rmv1a1.NonResourcePolicyRule{{
+		Verbs:           verbAll,
+		NonResourceURLs: []string{rmv1a1.NonResourceAll}}}
+)
+
+func prws(subjects []rmv1a1.Subject, nrr []rmv1a1.NonResourcePolicyRule, rr ...rmv1a1.ResourcePolicyRule) rmv1a1.PolicyRulesWithSubjects {
+	return rmv1a1.PolicyRulesWithSubjects{
+		Subjects:         subjects,
+		ResourceRules:    rr,
+		NonResourceRules: nrr,
+	}
+}
+
+// All the particular usernames available from the controller manager
+var _ = []string{
```

Comment:
i replaced hard-coded list of controllers by "*" service-accounts from kube-system namespace in https://github.com/kubernetes/kubernetes/pull/85268. the former is fragile for maintain

Reply:
Yes.  But rather than delete this list, I kept it around in case we later decide we want to use it.

Reply:
I deleted the default-objects.go file, in favor of updating and using the default.go file.